tests/infra_test.go

```diff
@@ -909,6 +912,15 @@ var _ = Describe("[Serial]Infrastructure", func() {
 	})
 })
 
+func checkError(err error) error {
+	var e *neturl.Error
+	if errors.As(err, &e) && e.Timeout() {
```

Comment:
don't you mean to check that `&& err.Timeout()`? how can a an error you just created `e` have timeout?

Reply:
e is populated by `As`.

Reply:
i didnt created it, i just casted it, and it does have a timeout field, if timeout == true its the expected good error
else its a "bad error" such as `connect: network is unreachable` which we do have on ipv6